tf-modules/cumulus-rds-tf/variables.tf

```diff
@@ -89,3 +90,33 @@ variable "engine_version" {
   type        = string
   default     = "10.12"
 }
+
+variable "max_capacity" {
+  type = number
+  default = 4
+}
+
+variable "min_capacity" {
+  type = number
+  default = 2
+}
+
+### Required for user/database provisioning
+variable "prefix" {
+  type = string
+}
+variable "provision_user_database" {
+  description = "true/false flag to configure if the module should provision a user and database using default settings"
+  type = bool
+  default = false
+}
+
+variable "permissions_boundary_arn" {
+  type    = string
+  default = ""
+}
+
+variable "rds_user_password" {
+  type    = string
+  default = "changeme"
```

Comment:
Should we have a default for this? It seems better to force users to set a value?

Reply:
Hmm, possibly.     Or perhaps a generated value.     The issue here is that to some degree it may be confusing to have to set this if `provision_user_database` is set to `false`

Reply:
I see. A generated value also seems logical to me then

Reply:
How does this feel: 79ad4c74f

Reply:
79ad4c74f